packages/ui-causal-graph-editor/src/shared/rendering/node/node-object.tsx

```diff
@@ -239,14 +240,14 @@ export class NodeObject extends PIXI.utils.EventEmitter<(typeof MOUSE_EVENTS)[nu
                 const textStyle = getTextStyle(nodeStyle.label_size);
                 const trimmedText = trimToFit(nodeStyle.label, maxSize, textStyle);
 
-                const txt = new PIXI.Text(trimmedText, textStyle);
-                txt.resolution *= 2;
+                const txt = new PIXI.Text({ text: trimmedText, style: textStyle });
+                txt.resolution = 2;
```

Comment:
Was it always 1 before? I remember in some cases some resolution numbers were different depending on the display (retina vs other), there might have been a reason we were multiplying here